example/meson.build

```diff
@@ -15,13 +15,6 @@ print(cysignals.__file__.replace('__init__.py', ''))
 ).stdout().strip()
 cysignals = declare_dependency(include_directories: inc_cysignals)
 
-# Make declarations in Cython code available to C include files
```

Comment:
As far as I know, meson does not understand `# cython:` directives.  Anyway, there is no need to touch `meson.build` files in this PR.

Reply:
I also thought so but Cython actually still looks for these magic comments. So it's okay to remove change the meson files. What is definitely no longer working are `#distutils` comments.

Reply:
If you think about it it makes sense — you need to give meson precisely the information needed for it to know which commands should be run etc.

So e.g. `language=c++` is needed, so that meson knows it should run `gcc a.cpp` instead of `gcc a.c`. Dependencies specification is needed so that meson knows it should rebuild the file when dependency version is updated.

The late include flag… probably not so much.

Reply:
(for future reference:)

https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html#globally

The cython compiler takes the directives straight from the pyx/pxd and/or command line. Because it's now placed in the file, you don't have to pass it (via meson) to the `cython` command (although meson *can* override compiler directives in the file by adding project arguments).